### ISSUE TITLE
Healtcheck fail url is not triggered

### ISSUE THREAD

**Issue by l4t3b0** [pfidr34/docker-rclone#15]
Hi pfidr34

I would like to ask for your help to solve a problem.

I do believe that the healthcheck fail url is not triggered everytime it should be.

My situation is the following:
I have configured a webdav link to a sharepoint url.
There are some nasty files in the Forms directory that has been generated by the sharepoint.
Because of these files under the Forms directory i have problems running the container.

I can easily overcome the problem with the --exclude /Forms/** parameter and than it works fine.
My problem is that why isnt the fail url of the healtchecksio is not triggered.

My log is:

```
INFO: Running rclone v1.51.0
INFO: Cron shortcut @weekly re-written to 0 0 * * 0
INFO: Starting sync.sh pid 28 Mon May 18 17:40:44 CEST 2020
INFO: Removing logs older than 120 day(s)...
INFO: Source directory is not empty and can be processed without clear loss of data
INFO: Log file output to /logs/2020_05_18-17_40_52.txt
INFO: Sending start signal to healthchecks.io
--2020-05-18 17:40:52-- https://hc-ping.com/6c033264-....7de3584/start
Resolving hc-ping.com... 176.9.71.146, 178.63.26.145, 2a01:4f8:151:18c::2, ...
Connecting to hc-ping.com|176.9.71.146|:443... connected.
HTTP request sent, awaiting response... 200 OK
Length: 2 [text/plain]
Saving to: '/dev/null'

0K 100% 368K=0s

2020-05-18 17:40:52 (368 KB/s) - '/dev/null' saved [2/2]

INFO: Starting rclone sync webdav-alapitvany-documents:/Megosztott dokumentumok/ /data --config /config/rclone.conf -v --log-file=/logs/2020_05_18-17_40_52.txt
```

The rclone log file content is:
```
2020/05/18 17:41:17 ERROR : Forms/Upload.aspx: corrupted on transfer: sizes differ 6076 vs 93436
2020/05/18 17:41:17 INFO  : Forms/Upload.aspx: Removing failed copy
2020/05/18 17:41:18 ERROR : Forms/repair.aspx: corrupted on transfer: sizes differ 3292 vs 150156
2020/05/18 17:41:18 INFO  : Forms/repair.aspx: Removing failed copy
2020/05/18 17:41:22 INFO  : Local file system at /data: Waiting for checks to finish
2020/05/18 17:41:22 INFO  : Local file system at /data: Waiting for transfers to finish
2020/05/18 17:41:22 ERROR : Local file system at /data: not deleting files as there were IO errors
2020/05/18 17:41:22 ERROR : Local file system at /data: not deleting directories as there were IO errors
2020/05/18 17:41:22 ERROR : Attempt 3/3 failed with 7 errors and: corrupted on transfer: sizes differ 3292 vs 150156
2020/05/18 17:41:22 Failed to sync with 7 errors: last error was: corrupted on transfer: sizes differ 3292 vs 150156
```

It is important to note, that it is not only i dont get the fail url triggered, but the docker container simply stops. :(

If you have difficulties reproducing the situation than im gladly willing to run some debuging version of the sync.sh for you.

Thanks for your help in advance

**Comment by pfidr34** [comment]
I bet something is making rclone hard crash the container prior to getting a return code that we use to alert healthchecks.io at the bottom of sync.sh

I'll try to replicate this week.

**Comment by pfidr34** [comment]
@l4t3b0 thanks again for submitting another issue. This is resolved under commit [5b4abe8](https://github.com/pfidr34/docker-rclone/commit/5b4abe8bfda8346506ff03279529bec173907591).

A new build has been pushed to Docker Hub.
